bundles/aliyun/src/main/java/org/apache/gravitino/oss/credential/OSSTokenProvider.java

```diff
@@ -166,6 +168,7 @@ private String createPolicy(Set<String> readLocations, Set<String> writeLocation
                       Statement.builder()
                           .effect(Effect.ALLOW)
                           .addAction("oss:GetBucketLocation")
+                          .addAction("oss:GetBucketInfo")
```

Comment:
Please confirm that it is okay to have two actions added here, and update the comments accordingly.

Reply:
it's ok to do so, I prefer not adding extra comments because this pattern is used several time in the class, it's self explained. I add extra comment for why adding the action.